### PR TITLE
Add admin to all orgs

### LINKED ISSUE #7045: Fix Grafana access for KKP Admins
KKP Admin users should be able to access data of all organizations (= KKP Projects) in Grafana UI.

### PR DESCRIPTION
**Pull request by aborilov**
**What this PR does / why we need it**:
In this PR we are taking into the account isAdmin flag of KKP user. We are adding KKP admins to all grafana organizations

**Which issue(s) this PR fixes** *(optional, in `fixes #<issue number>(, fixes #<issue_number>, ...)` format, will close the issue(s) when PR gets merged)*:
Fixes #7045 

**Special notes for your reviewer**:

**Documentation**:

**Does this PR introduce a user-facing change?**:

```release-note
NONE
```

### CODE REVIEW

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller.go [hunk 1]
```diff
@@ -24,6 +24,7 @@ import (
 
 	"go.uber.org/zap"
 
+	"github.com/grafana/grafana/pkg/models"
```
Comment (jiachengxu) [inline pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: Please sort the imports

File: pkg/controller/seed-controller-manager/mla/user_grafana_controller.go [hunk 2]
```diff
@@ -25,11 +25,13 @@ import (
 
 	"go.uber.org/zap"
 
+	"github.com/grafana/grafana/pkg/models"
```
Comment (jiachengxu) [inline pkg/controller/seed-controller-manager/mla/user_grafana_controller.go]: Please sort the imports

Comment (aborilov) [reply pkg/controller/seed-controller-manager/mla/user_grafana_controller.go]: done

File: pkg/controller/seed-controller-manager/mla/org_grafana_controller.go (hunk 1 above)
Comment (aborilov) [reply pkg/controller/seed-controller-manager/mla/org_grafana_controller.go]: done

**Pull request merged**